### ISSUE TITLE
Segfault while running websocket server

### ISSUE THREAD

**Issue by bsansouci** [oven-sh/bun#12721]
### How can we reproduce the crash?

Not much to report, this has just been running for a couple of days managed by PM2 but not through the cluster, just listening to same port.

### Relevant log output

```shell
Bun v1.1.20 (ae194892) Linux x64
Linux Kernel v6.1.0 | glibc v2.36
CPU: sse42 popcnt avx avx2 avx512
Args: "/home/[REDACTED]/.bun/bin/bun" "run" "/home/[REDACTED]"
Features: jsc fetch(3533) http_server transpiler_cache(20) tsconfig_paths tsconfig
Builtins: "bun:main" "node:buffer" "node:crypto" "node:events" "node:fs" "node:http" "node:https" "node:net" "node:os" "node:path" "node:stream" "node:string_decoder" "node:tls" "node:tty" "node:url" "node:util" "node:util/types" "node-fetch"
Elapsed: 53040246ms | User: 2266000ms | Sys: 1881729ms
RSS: 1.07GB | Peak: 0.29GB | Commit: 1.07GB | Faults: 0

panic: Segmentation fault at address 0x27C84287060
oh no: Bun has crashed. This indicates a bug in Bun, not your code.

To send a redacted crash report to Bun's team,
please file a GitHub issue using the link below:

https://bun.report/1.1.20/lr1ae19489AigihwD2v7nnE+kgP+vi7uDinrnwD0t8q/DsiuzvDghk0rDmzoiBA24nBh6n+67D
```
```


### Stack Trace (bun.report)

Bun v1.1.20 ([`ae19489`](<https://github.com/oven-sh/bun/tree/ae19489250ab74ee3ea7d5b50fca5d7d2b4e6d66>)) on linux x86_64 [RunCommand]

Segmentation fault at address 0x27C84287060

- [`ld-temp.o:0`](<https://github.com/oven-sh/bun/blob/ae19489250ab74ee3ea7d5b50fca5d7d2b4e6d66/ld-temp.o#L0>): `WTF::Unicode::convertReplacingInvalidSequences`
- `??`
- [`http.zig:2570`](<https://github.com/oven-sh/bun/blob/ae19489250ab74ee3ea7d5b50fca5d7d2b4e6d66/src/http.zig#L2570>): `src.http.onWritable__anon_151239`
- [`uws.zig:881`](<https://github.com/oven-sh/bun/blob/ae19489250ab74ee3ea7d5b50fca5d7d2b4e6d66/src/deps/uws.zig#L881>): `src.deps.uws.NewSocketHandler`
- [`epoll_kqueue.c:399`](<https://github.com/oven-sh/bun/blob/ae19489250ab74ee3ea7d5b50fca5d7d2b4e6d66/src/eventing/epoll_kqueue.c#L399>): `us_poll_change`
- [`array_hash_map.zig:2022`](<https://github.com/oven-sh/bun/blob/ae19489250ab74ee3ea7d5b50fca5d7d2b4e6d66/src/deps/zig/lib/std/array_hash_map.zig#L2022>): `src.http.HTTPThread.processEvents`
- [`string_immutable.zig:885`](<https://github.com/oven-sh/bun/blob/ae19489250ab74ee3ea7d5b50fca5d7d2b4e6d66/src/string_immutable.zig#L885>): `src.http.NewHTTPContext`
- `??`

<!-- from bun.report: kJwF6GydPS3xhlVhdO7q -->

**Comment by cirospaciari** [comment]
Fixed in canary v1.1.22 if is happening after `bun upgrade --canary` feel free to reopen

**Comment by bsansouci** [comment]
Thank you!
